### PR TITLE
Fix new installer test

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

File: tests/integration/tests.mk [hunk 1]
```diff
@@ -64,7 +64,7 @@ NEW_INSTALLER_TARGETS = $(shell GOPATH=${GOPATH} go list ../istio/tests/integrat
 # Runs tests using the new installer. Istio is deployed before the test and setup and cleanup are disabled.
 # For this to work, the -customsetup selector is used.
 test.integration.new.installer: | $(JUNIT_REPORT)
-	KUBECONFIG=${INTEGRATION_TEST_KUBECONFIG} kubectl apply -k github.com/istio/installer/crds
+	KUBECONFIG=${INTEGRATION_TEST_KUBECONFIG} kubectl apply -k github.com/istio/installer/kustomize/cluster
```
Comment (nmittler) [inline tests/integration/tests.mk]: "kustomize"?

**Comment by howardjohn** [comment]
Kustomize is basically the official kubectl way to customize templates,
somewhat like helm

**Pull request merged**